src/Prelude/Linear.hs

```diff
@@ -50,6 +49,8 @@ infixr 0 $
 const :: a ->. b -> a
 const x _ = x
 
+-- In `base`, this is not a prelude function (it's in `Data.Tuple`), maybe we
```

Comment:
This comment seems out-of-place. It's neither connected to the goal of this PR nor actionable because not tagged with `TODO` or `XXX`.

Reply:
It's just something I noted during this PR. As part of the warning hunting bit.